index.html

```diff
@@ -68,87 +75,94 @@ <h1>Alex Hamn</h1>
           <li>
             <a href="#"><img src="media/icons/m.svg" alt="m" /></a>
           </li>
+          <li>
+            <h4>↓ Work</h4>
+          </li>
         </ul>
         <h4>↓ Work</h4>
       </section>
       <section class="grid projects">
         <div>
           <h2>
-            MY RECENT <br />
-            WORK
+            MY RECENT WORK
           </h2>
         </div>
         <article class="container project">
-          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder"
-          />
-          <h3>Multi-Post Stories</h3>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="container languages">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button>See Project</button>
+          <img src="https://picsum.photos/684/448?grayscale" alt="placeholder"/>
+          <div class="container project-text">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text ever
+              since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
```

Comment:
Increase the width for each one of the projects paragraphs

Reply:
It seems like all the project paragraphs are of an adequate width; they're even a few pixels larger than the original template. 😄 
So, could you please explain to me what it is exactly what you mean by this comment? 🙏🏼

Reply:
Each one of the projects has a paragraph on the side you need to increase the width to reduce the empty space a little bit

Reply:
Got it, ty 🙏🏼 😁